fix_ascendancy_positions.py

```diff
@@ -120,6 +120,20 @@ def fix_ascendancy_positions(path: os.PathLike) -> None:
             if node["Node"]["name"] in EXTRA_NODES_STATS:
                 data["nodes"][node["Node"]["skill"]]["stats"] = EXTRA_NODES_STATS[node["Node"]["name"]]["stats"]
                 data["nodes"][node["Node"]["skill"]]["reminderText"] = EXTRA_NODES_STATS[node["Node"]["name"]]["reminderText"]
+    
+    # Remove unused image zoom levels and data around them
+    for sprite in data["sprites"]:
```

Comment:
Correcting my comment...
for sprite,sprite_data in data["sprites"].items():
    if "1" in sprite_data:
        data["sprites"][sprite] = sprite_data["1"]
    if "0.3835" in sprite_data:
        data["sprites"][sprite] = sprite_data["0.3835"]